docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -443,6 +443,38 @@ paths:
           description: Group not found
           schema:
             $ref: '#/definitions/Error'
+    post:
+      tags:
```

Comment:
Nice job picking up on this `tags` thing! It's specific to the documentation generator we're using.